triton/apps/generative_ai_tools.rst

```diff
@@ -0,0 +1,17 @@
+Generative AI Tools on Triton
+=============================
+Speech to Text: Whisper
+----------------------------
+Whisper is a general-purpose speech recognition model. It is a multitasking model that can perform multilingual speech recognition, speech translation, and language identification. For the usage on Triton, please check out the `whisper <https://scicomp.aalto.fi/triton/apps/whisper/>`__ page.
```

Comment:
```suggestion
Whisper is a general-purpose speech recognition model. It is a multitasking model that can perform multilingual speech recognition, speech translation, and language identification. For the usage on Triton, please check out the :doc:`whisper <whisper>` page.
```
Internal links can be made this way, and it'll know when it breaks.  This is a relative path but also thinsg like /triton/apps/whisper or ../some-dir/page work.